packages/api-generator/src/template.d.ts

```diff
@@ -1,5 +1,5 @@
 import type { AllowedComponentProps, ComponentPublicInstance, FunctionalComponent, RenderFunction, VNodeChild, VNodeProps } from 'vue'
-import type { __component__ } from '@/__name__'
+import type { __component__ } from '@/components'
```

Comment:
@KaelWD can you double check your change? I've reverted it here as with this change it didn't generate anything for me, ex. `api-generator/dist/attributes.json` was empty (so nothing related to this PR)

Reply:
Looks fine to me on `next`